### ISSUE TITLE
Unitary decomposition missing from the pre-built wheels

### ISSUE THREAD

**Issue by DevelopDaily** [QuTech-Delft/OpenQL#343]
This will fail:

```
from openql import openql as ql
u1 = ql.Unitary("U_name", [0.5+0.5j,0.5-0.5j,0.5-0.5j,0.5+0.5j])

```
The error message is:

`AttributeError: module 'openql.openql' has no attribute 'Unitary'`

That is copied from the documentation. The OpenQL is installed from the pre-built wheels like this:

`pip install qutechopenql`

Is the pre-built stuff up-to-date?

**Comment by jvanstraten** [comment]
It's actually the documentation that's running way ahead of the pre-built stuff, because the documentation didn't exist at all yet when the previous version was released. We're working on stabilizing the current develop branch to bring out a new release ASAP, however, and then we'll probably have ReadTheDocs track `master` instead. Or, in my opinion, we should just release more often now that the process is automated.

**Comment by wvlothuizen** [comment]
Solved in 0.10.0
